.github/workflows/smoke-checks.yml

```diff
@@ -10,6 +10,10 @@ on:
   release:
     types:
       - created
+
+concurrency: 
+  group: ${{ github.head_ref }}
+  cancel-in-progress: true
```

Comment:
I know we are not using them right now, but maybe let's add this to the full checks as well? 🤔

Reply:
Full checks are not run on pushes to main, this is not required for it. same applies to testflight